src/main/java/teammates/ui/controller/InstructorEditStudentFeedbackSaveAction.java

```diff
@@ -69,14 +70,12 @@ protected void checkAdditionalConstraints() {
             boolean isRecipientVisibleToInstructors = questionAttributes.showRecipientNameTo.contains(FeedbackParticipantType.INSTRUCTORS);
             boolean isResponseVisibleToInstructors = questionAttributes.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS);
             
-            
-            if (!isGiverVisibleToInstructors || !isRecipientVisibleToInstructors || !isResponseVisibleToInstructors) {
+            if (!isResponseVisibleToInstructors || !isGiverVisibleToInstructors || !isRecipientVisibleToInstructors) {
```

Comment:
Doing response first we will have better coverage due to not short circuiting and also due to the nature of what is a possible setting (responseVisible == false means the other two is automatically false too)